neqo-transport/src/cc/mod.rs

```diff
@@ -0,0 +1,33 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Congestion control
+#![deny(clippy::pedantic)]
+
+pub mod cc;
```

Comment:
I think that you should move the interface definition into this file.   Otherwise we have neqo_transport::cc::cc::CongestionControl as the canonical name for the interface, which is a little weird.